corgi/core/fixups.py

```diff
@@ -94,3 +94,22 @@
 def cpe_lookup(product_stream_name: str) -> set[str]:
     """Manual CPE overrides for streams which cannot be matched with variants"""
     return set(supported_stream_cpes.get(product_stream_name, []))
+
+
+external_names = {
+    # Can't parse the stream version properly
+    "openstack-13-els": "RHEL-7-OS-13-ELS",
+    "rhel-m-4.4": "RHEL-8-RHEV-4",
+}
+
+
+def external_name_lookup(product_stream_name: str) -> str:
+    stream_prefixes = ("dts-", "rhel-br-", "mtr-", "quay-")
+    # 'dts' streams share variants with 'rhscl', however the component sets are different
+    # 'rhel-br' streams share variants with 'rhel' however the components sets are different
+    # 'mtr-' streams share the 'RHEL-8-MTR-1' variant, but have distinct brew_tags
+    # 'quay-' streams share the 'QUAY-3-RHEL-8' variant, but have distinct brew_tags
+    for stream_prefix in stream_prefixes:
+        if product_stream_name.startswith(stream_prefix):
```

Comment:
This assumes the product_stream_name is always lowercase. I hope that's always the case in prod-defs, but maybe for safety we could do below:
```suggestion
        if product_stream_name.lower().startswith(stream_prefix):
```

Reply:
Updated.